src/voice/connection.rs

```diff
@@ -267,6 +310,13 @@ impl Connection {
             self.udp.send_to(&bytes, self.destination)?;
         }
 
+        // Reconfigure encoder bitrate.
+        // From my testing, it seemed like this needed to be set every cycle.
+        // -- FelixMCFelix
+        match self.encoder.set_bitrate(bitrate) {
+            Ok(_) => {},
+            Err(e) => {println!("Bitrate set unsuccessfully: {:?}", e);},
```

Comment:
- this could be simplified to `Err(e) => println!( ...` (except for the above comment about using `if-let` syntax)
- libraries shouldn't `println`, let's use `warn!` in this case since it's an error but not fatal